sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/simple/TrigonometryTests.java

```diff
@@ -0,0 +1,89 @@
+package org.dbsp.sqlCompiler.compiler.sql.simple;
+
+import org.dbsp.sqlCompiler.compiler.sql.SqlIoTest;
+import org.junit.Test;
+
+public class TrigonometryTests extends SqlIoTest {
+    @Test
+    public void testSin() {
+        this.qs(
+                "SELECT sin(0);\n" +
```

Comment:
All these values are decimal values.
You probably should write the same test with double values as arguments.
And replace 3.14 with pi when we have implemented that.
Maybe you can push another commit in this PR for PI.

Reply:
you should also document where the tests are coming from. For example "tested using Postgres".